docs/source/federation-version-support.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Federation Version Support in the Apollo Router
+title: Federation Version Support in the GraphOS Router
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page)